target/linux/lantiq/files-5.4/arch/mips/boot/dts/lantiq/vr9_avm_fritz7360v2.dts

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "vr9_avm_fritz736x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7360v2", "avm,fritz736x", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7360 V2";
+};
+
+&power_green {
+	label = "fritz7360v2:green:power";
+};
+
+&power_red {
+	label = "fritz7360v2:red:power";
+};
+
+&info_green {
+	label = "fritz7360v2:green:info";
+};
+
+&wifi {
+	label = "fritz7360v2:green:wlan";
+};
+
+&info_red {
+	label = "fritz7360v2:red:info";
+};
+
+&dect {
+	label = "fritz7360v2:green:dect";
+};
+
+&state_default {
+	pcie-rst {
+		lantiq,pins = "io38";
```

Comment:
Can you try using io21 here? It might fix the irq problem.

Reply:
I have tried io21 and I see one difference in dmesg.

io38:
`[    1.066576] pci 0000:01:00.0: [168c:ff1c] type 00 class 0x020000`
io21:
`[    1.068691] pci 0000:01:00.0: [168c:002e] type 00 class 0x028000`

So io21 is the right one?

Reply:
Interesting, I thought only the message `irq 144: nobody cared (try booting with the "irqpoll" option)` would change. In your earlier dmesg the wifi card was detected as 168c:002e. Do you rebooted multiple times to get 168c:ff1c? Or was it a boot from power off.

How do you test it? Do you only booted once or rebooted multiple times?

So far, atm I would like to use io21.

Reply:
First boot after flashing I get 168c:ff1c with io38 and 168c:002e with io21.
After a reboot io38 boots directly with 168c:002e.

Reply:
Thanks for looking into it even more! It looks like, the pcie is not a good indicator for it. I would guess the different pci ids comes from the calibration data, which got loaded when OpenWrt boots.